### DISCUSSION TITLE
Security - 2

### DISCUSSION THREAD

**Discussion by changjurhee** [changjurhee/AONE#12] [Polls]
## Quality Attributes

Type | Description | QA Scenario (Stimulus / Source of stiulus / Artifact / Environment / Response / Response measure) | Tatics
 -- | -- | -- | --
Security | System shall transfer data for sessions, video data, audio streaming data between server & client in secure way. | Even if communication data between server and client is leaked by an outsider, communication must be encrypted so that security information is not exposed.<br>- Data is leaked to the outside <br>- Outsider<br>- Security information<br>- All<br>- Communication data encryption<br>- Unknown raw security information |

**Comment by BrunoYoo** [comment]
서버와 주고 받는 데이터가 대략, 아래 정도일것 같은데 세가지 모두 TLS (?) 로 송수신하면 만족하지 않을까요?
- 사용자 정보, (등록, 로그인, 정보 변경, Contact 정보(검색결과))
- Call session 정보, 
- Video / Audio data

**Comment by changjurhee** [comment]
- gstream (SRTP, RTCP)
- UDP
